tests/spec/fs.symlink.spec.js

```diff
@@ -44,3 +44,33 @@ describe('fs.symlink', function() {
     });
   });
 });
+
+describe('fs.promises.symlink', function(){
+  beforeEach(function(done){
+    util.setup(function(){
+      var fsPromises = util.fs().promises;
```

Comment:
You can get rid of all this code, and just do `beforeEach(util.setup)`.  We don't need to do anything with `fs.symlink` before each test is run, only during the tests.